front/app/components/FilterBoxes/utils.tsx

```diff
@@ -0,0 +1,11 @@
+export const scrollToTopIdeasList = () => {
+  const ideasListPanel = document.getElementById('ideas-list-scroll-anchor');
+  const boundingBox = ideasListPanel?.getBoundingClientRect();
```

Comment:
Why did you add boundingBox, btw? :) If we're not using its value, we can probably remove it?

Reply:
Ah - yes that was remaining from the previous solution. I'll remove it :+1: Thanks for flagging!